modules/mono/glue/cs_files/Color.cs

```diff
@@ -290,7 +290,7 @@ public int to_32()
             return c;
         }
 
-        public int to_ARGB32()
+        public int ToArgb32()
```

Comment:
Here `ToARGB32`, below `ToHTML`.. Or what would be the convention in such cases?

Reply:
Yeah, I wasn't sure about these either.
The way it is now is using the default visual studio/resharper naming convention.

Reply:
It looks like the style you used is the recommended one: https://msdn.microsoft.com/en-us/library/ms229043%28v=vs.100%29.aspx#Capitalization%20Rules%20for%20Acronyms